src/desktop/apps/auction_support/routes.coffee

```diff
@@ -7,6 +7,8 @@ registerAndRedirect = (sale, req, res, next) ->
   req.user.fetchCreditCards
     error: res.backboneError
     success: (creditCards) ->
+
+      # FIXME: What happens when creditCards.length === 0? Noticing that the app hangs
       if (creditCards.length > 0)
```

Comment:
This is the bug -- the app will hang if a user has no credit cards as the `else` case hasn't been handled.